subiquity/server/controllers/filesystem.py

```diff
@@ -288,6 +289,11 @@ async def configured(self):
         self._configured = True
         if self._info is None:
             self.set_info_for_capability(GuidedCapability.DIRECT)
+        if (
+            self.app.base_model.source.search_drivers
+            is SEARCH_DRIVERS_AUTOINSTALL_DEFAULT
+        ):
+            self.app.base_model.source.search_drivers = not self.is_core_boot_classic()
```

Comment:
This ends up kind of a weird default.  I think we should note it in the autoinstall reference.
The reference will need update anyhow for core boot but I don't want to forget this part.

Reply:
Yeah this behaviour is a bit strange. I've added a note.

Reply:
Thank you, I like the doc blurb